faqs/galaxy/datasets_custom_genomes.md

```diff
@@ -0,0 +1,20 @@
+---
+title: How to use Custom Reference Genome?
+area: datasets
+box_type: tip
+layout: faq
+contributors: [jennaj, Nurzhamalyrys]
+---
+
+
+- In Galaxy, a custom reference genome, which is a dataset from the history loaded by users, is a FASTA formatted dataset. It can be used in place of a native reference genome, that is local or cached by administrators, with most tools.
+- A reference genome contains the nucleotide sequence of the chromosomes, scaffolds, transcripts, or contigs for single species. It is representative of a specific genome build or release.
```

Comment:
This is a bit confusing to me. What about something like: 
> A reference genome is...
> You have two options to use reference genomes in Galaxy: _native_ (provided by the server administrators and used by most of the tools) and _custom_ (uploaded by users in FASTA format). This FAQ covers the latter.

I'm not an expert on this, so don't take it too seriously :)

Reply:
> This is a bit confusing to me. What about something like:
> 
> > A reference genome is...
> > You have two options to use reference genomes in Galaxy: _native_ (provided by the server administrators and used by most of the tools) and _custom_ (uploaded by users in FASTA format). This FAQ covers the latter.
> 
> I'm not an expert on this, so don't take it too seriously :)

Hi! 
Yeah, your version is easier to understand, so changed to your version. Please, let me know if something should be changed also.